src/vnsw/agent/cmn/agent.cc

```diff
@@ -470,7 +473,7 @@ static bool interface_exist(string &name)
 	bool ret = false;
 	string tname = "";
 
-	ifs = if_nameindex();
+	ifs = osspecific_if_nameindex();
```

Comment:
As per our Tuesday discussion, we should avoid breaking compatibility if possible.